standalone_self_upgrade.cf.in

```diff
@@ -234,6 +246,12 @@ bundle agent cfengine_software_version
       "Manage CFEngine Version"
         usebundle => cfengine_software_version_packages1;
 
+      # TODO, remove this and cfe_internal/enterprise/windows_unattended_upgrade.cf
+      # when ENT-6823 allows us to use msiexec.bat packages module.
+      "Windows Unattended Upgrade Workaround"
+        usebundle => windows_unattended_upgrade,
+        if => "windows";
```

Comment:
i might have put the version comparison check here to avoid calling the bundle if already at target version.

Reply:
Right. Much of this is copied from an earlier iteration that you worked on I thought. So I like that change as well. Will try it out.